modules/nf-core/gemma/kinshipmatrix/environment.yml

```
@@ -0,0 +1,7 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/environment-schema.json
+channels:
+  - conda-forge
+  - bioconda
+dependencies:
+  - "bioconda::gemma=0.98.5"
```

You don't need here the `"`
```suggestion
  - bioconda::gemma=0.98.5
```